src/backend/clearboard/main.py

```diff
@@ -47,3 +96,124 @@ async def post_picture(file: UploadFile = File(...)):
         with open(f"{path}/{file.filename}", 'wb') as f:
             shutil.copyfileobj(file.file, f)
         return {"message": file.filename}
+
+
+def image_to_base64(img: np.ndarray) -> bytes:
+    """ Given a numpy 2D array, returns a JPEG image in base64 format """
+    img_buffer = cv2.imencode('.jpg', img)[1]
+    return base64.b64encode(img_buffer).decode('utf-8')
+
+
+def get_image(image):
+    return image_to_base64(image)
+
+
+@app.get("/process")
+async def photo(roomName: str, process: str):
+    try:
+        img_cropped_path = './' + roomName + '/' + roomName + 'cropped.jpg'
+        original_img_path = './' + roomName + '/' + roomName + '.jpg'
+        processed_img_path = './' + roomName + '/' + roomName + process + ".jpg"
+        if os.path.exists(os.path.abspath(img_cropped_path)):
+            img_to_process = img_cropped_path
+        elif os.path.exists(os.path.abspath(original_img_path)):
+            img_to_process = original_img_path
+
+        if process == 'Color':
+            color.whiteboard_enhance(cv2.imread(
+                img_to_process), processed_img_path)
+        elif process == 'B&W':
+            blackNwhite.black_n_white(img_to_process, processed_img_path)
+        elif process == 'Contrast':
+            contrast.enhance_contrast(img_to_process, processed_img_path)
+        elif process == 'original':
+            processed_img_path = img_to_process
+        elif process == 'SuperRes':
+            super_res.super_res(img_to_process, processed_img_path, "./clearboard/EDSR/EDSR_x4.pb")
+        else:
+            processed_img_path = img_to_process
+        img = cv2.imread(processed_img_path)
+        volume = np.asarray(img)
+        image = get_image(volume)
+        print('image sent')
+        return Response(content=image)
+
+    except:
```

Comment:
try/except without a precise error is an anti-pattern in Python. You should know what error you expect or let it fail with a 500 so that we are aware of the error and can find a solution to it.